index.bs

```diff
@@ -3666,6 +3684,21 @@ credential.
         When storing a {{PublicKeyCredentialUserEntity/displayName}} member's value,
         the value MAY be truncated as described in [[#sctn-strings-truncation]]
         using a size limit greater than or equal to 64 bytes.
+
+    :   <dfn>language</dfn>
+    ::  An OPTIONAL [=resource-wide default=] language field [[!String-Meta]] for the entity.
+
+        The [=Client=] SHOULD verify that the value is a [=well-formed language tag=].
+
+        This member MAY be ignored if the combination of [=client=] and [=authenticator=] cannot suitably process and store it.
+
+    :   <dfn>direction</dfn>
+    ::  An OPTIONAL [=resource-wide default=] [=block direction=] field [[!String-Meta]] for the entity.
+
+        This SHOULD be one of the values `"ltr"`, `"rtl"` or `"auto"`.
```

Comment:
Shouldn't this SHOULD be MUST? There is no utility in allowing other strings here and you permit clients to ignore unknown values later.

Reply:
I made it like this for consistency with our other [enumerations as DOMString types](https://w3c.github.io/webauthn/#sct-domstring-backwards-compatibility), for example the [`attestation`](https://w3c.github.io/webauthn/#dom-publickeycredentialcreationoptions-attestation) parameter. "MUST" would rather contradict the later statement that unknown values are allowed but ignored, while "SHOULD" expresses some more flexibility in case new valid values are added in the future. That is perhaps unlikely for `direction`, but you never know - maybe a `"ttb-ltr"` value could at some point make sense for some Asian scripts, for example.

Reply:
No, that won't happen for direction. Vertical vs. horizontal is a presentational variation (often called the "writing mode"). Direction pertains the Unicode Bidi Algorithm and the values have been stable for a long time. But the consistency argument is reasonable.